docs/coding-style.md

```diff
@@ -27,7 +27,7 @@ Static fields and properties should be placed before instance ones.
 
```

Comment:
As there's this PR on the way, it's a good opportunity to address comments I've left on #6786 after it got merged.